bumble/hci.py

```diff
@@ -2694,17 +2709,45 @@ class HCI_Reset_Command(HCI_Command):
 
 # -----------------------------------------------------------------------------
 @HCI_Command.command(
-    [
+    fields=[
         ('filter_type', 1),
-        ('filter_condition', '*'),
-    ]
+        # TODO: separate this field into condition_type and condition fields
```

Comment:
Unfortunately, there's no "easy" way to do this just from the fields description, so this would have to be done in code rather than data, as class methods (as is done for a small number of other commands that also need some level of custom parsing/serializing)

Reply:
Updated the comments